<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convert</class>
 <widget class="QMainWindow" name="convert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_convert">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="frc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主界面</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="decimal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>请输入你要转换的分数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="decimal2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="frc_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>请输入你要转换的有限小数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
